src/web/etc.rs

```diff
@@ -76,6 +76,13 @@ impl<T> From<HashOf<T>> for SerScaleHex<Hash> {
         hash.into()
     }
 }
+// implementation for payload
```

Comment:
I think this comment is unnecessary.

You can either remove it, or make it doccomment (with `///`) and write some expanded explanation, what kind of payload do you mean. But for me it seems obvious.